packages/react-components/react-migration-v0-v9/src/components/List/ListItem/useListItemStyles.styles.ts

```diff
@@ -0,0 +1,89 @@
+import type { SlotClassNames } from '@fluentui/react-utilities';
+import { makeStyles, makeResetStyles, mergeClasses, shorthands } from '@griffel/react';
+import { createCustomFocusIndicatorStyle } from '@fluentui/react-tabster';
+import type { ListItemSlots, ListItemState } from './ListItem.types';
+import { tokens } from '@fluentui/react-theme';
+
+export const listItemClassNames: SlotClassNames<ListItemSlots> = {
+  root: 'fui-ListItem',
+  media: 'fui-ListItem__media',
+  header: 'fui-ListItem__header',
+  contentWrapper: 'fui-ListItem__contentWrapper',
+  headerMedia: 'fui-ListItem__headerMedia',
+  contentMedia: 'fui-ListItem__contentMedia',
+  endMedia: 'fui-ListItem__endMedia',
+};
+
+const useRootBaseStyles = makeResetStyles({
+  ...shorthands.padding(0, '10px'),
+  ...shorthands.margin(0),
+  textIndent: 0,
+  listStyleType: 'none',
+  ...createCustomFocusIndicatorStyle(
+    {
+      ...shorthands.outline(tokens.strokeWidthThick, 'solid', tokens.colorStrokeFocus2),
+      ...shorthands.borderRadius(tokens.borderRadiusMedium),
+    },
+    { selector: 'focus' },
+  ),
+});
+
+/**
+ * Styles for the root slot
+ */
+const useStyles = makeStyles({
+  contentWrapper: {
+    fontSize: tokens.fontSizeBase200,
+  },
+
+  // The content should go all the way to the end if the content media is not present
+  contentWrapperWithoutMedia: {
+    gridColumnEnd: 4,
+  },
+  truncate: {
+    ...shorthands.overflow('hidden'),
+    textWrap: 'nowrap',
+    textOverflow: 'ellipsis',
+  },
+  rootSelected: {
+    backgroundColor: tokens.colorNeutralBackground1Selected,
```

Comment:
The v0 list actually has inverted styles to this for selection - the selected item is actually white and the others are grey 🤷

![image](https://github.com/microsoft/fluentui/assets/20744592/111b72eb-f48b-4248-bb43-c3797190ef47)

Reply:
https://fluentsite.z22.web.core.windows.net/0.66.2/components/list/definition?showCode=false&showRtl=false&showTransparent=false&showVariables=false#types-selectable-controlled

Reply:
Yea you're right, I noticed this but forgot to update it.